tests/Doctrine/Tests/DBAL/Platforms/SQLServer2012PlatformTest.php

```diff
@@ -369,6 +370,9 @@ public function testModifyLimitQueryWithComplexOrderByExpression(): void
         self::assertEquals($sql, $expected);
     }
 
+    /**
+     * @throws DBALException
```

Comment:
Why?

Reply:
I believe this is from the revert commit, I didn't change this myself.